witwidget/notebook/jupyter/js/package.json

```diff
@@ -22,13 +22,11 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "file-loader": "^3.0.1",
```

Comment:
Do we use the file-loader anywhere else in the WIT code?

Reply:
nope just in the spot i removed